kodilla-intro/src/main/java/Agent.java

```diff
@@ -0,0 +1,24 @@
+public class Agent {
+
+    private String name;
+    private double age;
+    private double height;
+
+    public Agent(String name, double age, double height) {
+        this.name = name;
+        this.age = age;
+        this.height = height;
+    }
+
+    public void checkParameters() {
+        if(this.name == null) {
+            return;
+        }
+        if(this.age > 30 && this.height > 160) {
```

Comment:
Nie … to nie moze byc void 
Ty masz dac odpowiedz, czy Agent jest starszy niz, wyzszy niz … Tu nic nie sysoutujesz